client/components/aboutUs/InfoSection.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import {
+  SubInfo,
+  Logo,
+  InfoBox,
+  Headline,
+  Text,
+  GreenBar,
+  HeadingGroup,
+} from "./styles";
+
+import { ABOUT_DATA, HEADER_DATA } from "./Resources";
+
+const InfoSection = () => {
+  return (
+    <>
+      <SubInfo>
+        <Logo src="/Home.png" />
+        <InfoBox>
+          <HeadingGroup>
+            <GreenBar />
+            <Headline>{HEADER_DATA.title}</Headline>
+          </HeadingGroup>
+          <Text>{HEADER_DATA.text}</Text>
+        </InfoBox>
+      </SubInfo>
+
+      {/* looping data from Resources.js */}
+      {ABOUT_DATA.map(data => {
```

Comment:
This can probably be optimized a bit, as we're returning a conditional render below, even though the layout is the same.
If we define a variable here for the color / theme, then we can just use that variable as a prop below.

ex. variable: "const darkSection = data.id % 2 !== 0"
ex. prop: "<Text light={darkSection}>{data.text}</Text>"

Reply:
Hey @Soreg, Thanks for the feedback. Can you explain this a bit more.

Reply:
Hey @Soreg, no need now. I figured it out.
I have added prop that way removed the conditional render:
```
{ABOUT_DATA.map(data => {
        const darkSection = data.id % 2 !== 0;
        return (
          <SubInfo key={data.id} grey={darkSection}>
            {data.svg}
            <InfoBox>
              <HeadingGroup>
                <MidGreenBar />
                <Headline light={darkSection}>{data.title}</Headline>
              </HeadingGroup>
              <Text light={darkSection}>{data.text}</Text>
            </InfoBox>
          </SubInfo>
        );
      })}
```